interfaces/apparmor/spec.go

```diff
@@ -745,7 +746,7 @@ func emitEnsureDir(spec *Specification, ifaceName string, ensureDirSpec *interfa
 
 	// Create entries for the remaining directories after MustExistDir up to and including EnsureDir
 	for iter.Next() {
-		emit("  owner %s/ rw,", replacePrefixHome(iter.CurrentPathNoSlash()))
+		emit("  owner %s/ rw,", replacePrefixHome(iter.CurrentPath()))
```

Comment:
Since we call `CurrentPath` and add a slash in the format string, perhaps this should just call `CurrentPathPlusSlash`?